recipes/pyncview/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyncview" %}
+{% set version = "0.99.38" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/BoldingBruggeman/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f6228406e715823a340ba4ac9c3d329359aa7adb27885fa64765736db6b4d082
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - multiplot = pyncview.multiplot:main
+    - pyncview = pyncview.pyncview:main
+
+requirements:
+  build:
+  host:
+    - python >=3.8
+    - pip
+    - hatchling
+  run:
+    - python >=3.8
+    - netcdf4
+    - xmlplot >=0.9.28
+    - matplotlib
+    - pyqt
+
+test:
+  imports:
+    - {{ name }}
+  requires:
+    - pip
+  commands:
+    - pip check
+    - multiplot -h
+
+about:
+  home: https://github.com/BoldingBruggeman/{{ name }}
```

Comment:
We don't abuse the jinja templating b/c it makes the recipe unreadable. We prefer to have the actual name here and in the import section so people can read it and click on links.

Reply:
fixed in latest commit